src/lib/parts.ts

```diff
@@ -84,20 +91,20 @@ export class AttributeCommitter {
   commit(): void {
     if (this.dirty) {
       this.dirty = false;
-      this.element.setAttribute(this.name, this._getValue());
+      this.element.setAttribute(this.name, String(this._getValue()));
```

Comment:
don't wrap in `String()`. has a cost for values that are already strings

Reply:
True, done